openshift_cli_installer/manifests/clusters.example.yaml

```diff
@@ -104,3 +104,4 @@ clusters:
   worker_replicas: 2
   worker_flavor: m5.4xlarge
   worker_root_disk_size: 128
+  log-level: info # optional, default: "error", supported options are debug, info, warn, error
```

Comment:
`log_level`

Reply:
can the log level be set in `openshift_cli_installer/manifests/install-config-template.j2` ?

Reply:
log level is not needed in `install-config-template.j2`